GDAXSharp/WebSocket/WebSocket.cs

```diff
@@ -83,6 +83,14 @@ public void Stop()
             }
 
             stopWebSocket = true;
+            
+            var json = JsonConfig.SerializeObject(new TickerChannel
+            {
+                Type = ActionType.Unsubscribe
```

Comment:
cool!

Reply:
OK, will do.  I'll clean up the spacing and remove the unneeded condition, the test will have to changed then because it will fail when it see a dispose without checking for .not. open